sqlmesh/core/model/definition.py

```diff
@@ -103,11 +102,13 @@ class _Model(ModelMeta, frozen=True):
         post: Post-hooks to run after the model executes.
         expressions: All of the expressions between the model definition and final query, used for setting certain variables or environments.
         python_env: Dictionary containing all global variables needed to render the model's macros.
+        mapping_schema: The schema of table names to column and types.
     """
 
     expressions_: t.Optional[t.List[exp.Expression]] = Field(default=None, alias="expressions")
     python_env_: t.Optional[t.Dict[str, Executable]] = Field(default=None, alias="python_env")
     jinja_macros: JinjaMacroRegistry = JinjaMacroRegistry()
+    mapping_schema: t.Dict[str, t.Any] = {}
```

Comment:
Can we do better than `t.Any` here? We can't even tell whether this can be serialized or not.